guides/source/configuring.md

```diff
@@ -58,6 +58,8 @@ NOTE: If you need to apply configuration directly to a class, use a [lazy load h
 
 Below are the default values associated with each target version. In cases of conflicting values, newer versions take precedence over older versions.
 
+#### Default Values For Target Version 7.2
+
```

Comment:
Would it make sense to add a placeholder here?
```suggestion

No defaults defined yet.
```

Reply:
It might, but then we'd have to update the linter to take into account non-configurations values. I'm not sure that complexity is worth it since this is only likely to happen right after a release.